src/main/java/stirling/software/SPDF/model/ApplicationProperties.java

```diff
@@ -289,24 +289,7 @@ public String toString() {
                     + "]";
         }
 
-        public static class CustomApplications {
-            private boolean installBookAndHtmlFormats;
-
-            public boolean isInstallBookAndHtmlFormats() {
-                return installBookAndHtmlFormats;
-            }
-
-            public void setInstallBookAndHtmlFormats(boolean installBookAndHtmlFormats) {
-                this.installBookAndHtmlFormats = installBookAndHtmlFormats;
-            }
-
-            @Override
-            public String toString() {
-                return "CustomApplications [installBookAndHtmlFormats="
-                        + installBookAndHtmlFormats
-                        + "]";
-            }
-        }
+        public static class CustomApplications {}
```

Comment:
Is this still needed?

Reply:
Not needed but kept just if we want to add in future, but yes its un-used

Reply:
let me remove actually

Reply:
removed